freqtrade/fiat_convert.py

```diff
@@ -188,6 +191,10 @@ def _find_price(self, crypto_symbol: str, fiat_symbol: str) -> float:
         if not self._is_supported_fiat(fiat=fiat_symbol):
             raise ValueError('The fiat {} is not supported.'.format(fiat_symbol))
 
+        # No need to convert if both crypto and fiat are the same
+        if crypto_symbol == fiat_symbol:
```

Comment:
I personally would move this into the public method `find_price` as the private method `_find_price` is only called from there ... somewhere around line 131 maybe...?
as the result will always be 1.0 - it should return that ASAP.